test/ptc/system/system_test.clj

```diff
@@ -0,0 +1,86 @@
+(ns ptc.system.system-test
+  (:require [clojure.test :refer [deftest is testing]]
+            [clojure.set :as set]
+            [clojure.edn :as edn]
+            [ptc.start :as start]
+            [ptc.tools.cromwell :as cromwell]
+            [ptc.tools.wfl :as wfl]
+            [ptc.util.jms :as jms]
+            [ptc.integration.jms-test :as jms-test])
+  (:import [java.util UUID]))
+
+(def environment
+  (or (System/getenv "environment") "dev"))
+
+(def bucket
+  (or (System/getenv "ptc_bucket_name") "gs://dev-aou-arrays-input"))
```

Comment:
Might be better if we use CAPS for ENV VARs everywhere(like in start.clj) for consistency?

Reply:
Also, I got:
```
Exception: java.lang.Exception: ptc: 1 exit status from: ("gsutil" "cat" "gs://dev-aou-arrays-input/HumanExome-12v1-1_A/388089b4-a2ff-44e9-abf5-337d8b12cbc7/1/ptc.json") : CommandException: No URLs matched: gs://dev-aou-arrays-input/HumanExome-12v1-1_A/388089b4-a2ff-44e9-abf5-337d8b12cbc7/1/ptc.json
```
with the test, it seems this bucket does not have the expected files?